### PR TITLE
Fix some issues found when save bigdl model to tensorflow format file

### PR DESCRIPTION
**Pull request by yiheng**
## What changes were proposed in this pull request?
This PR fixes some issues when saving bigdl model to tensorflow format file
1. BatchNormalization without affine
2. SpatialConvolution without bias
2. SpatialConvolution with conv group
3. Support save Scale layer
4. Support save LRN layer
5. get padding information issue in pooling layer

## How was this patch tested?
unit test, manual test

### CODE REVIEW

**Comment by yangw1234** [comment]
LGTM
